refs/biblio.json

```diff
@@ -139,10 +139,11 @@
         "href": "http://www.atsc.org/cms/standards/a72/A72-Part-1-2014.pdf",
         "date": "18 February 2014"
     },
+    "AX API": {
```

Comment:
I just concluded that from running the tests locally. Not sure what Apple would like, but I'll remove the entry for now.